deploy/templates/default-managed-identity-azuredeploy.json

```diff
@@ -721,7 +722,7 @@
     },
     {
       "type": "Microsoft.KeyVault/vaults",
-      "apiVersion": "2016-10-01",
+      "apiVersion": "2021-04-01-preview",
```

Comment:
We shouldn't use preview APIs

Reply:
The documentation around enableRbacAuthorization mentioned its only in preview, but I just checked version 2019-09-01 and it seems to work with that too.